addons/test_mail/tests/test_mail_followers.py

```diff
@@ -11,6 +11,7 @@
 from odoo.addons.mail.models.mail_mail import _UNFOLLOW_REGEX
 from odoo.addons.mail.tests.common import MailCommon
 from odoo.exceptions import AccessError
+from odoo.tools.misc import limited_field_access_token
```

Comment:
Mmmh ?

Reply:
Original PR had diff on the file: https://github.com/odoo/odoo/pull/187799/files#diff-d347ab277f81b02418598e81ab51a504b062c073901f89c858b671d4db8cfc1b

I'll check what happened at rebase, either unnecessary import or missing change in file.

Reply:
`test_inbox_notification_follower` -> `test_inbox_unfollow_information`

The test is no longer ensuring the full correct payload is received, so I guess I can just remove the import, and hope we don't introduce bugs in the now no longer covered parts, as it was the only test covering the inbox flow data...